src/Writers/HTMLWriter.jl

```diff
@@ -160,18 +160,19 @@ end
 Constructs and writes the page referred to by the `navnode` to `.build`.
 """
 function render_page(ctx, navnode)
-    @tags html body
+    @tags html body div
 
     page = getpage(ctx, navnode)
 
     head = render_head(ctx, navnode)
+    topbar = render_topbar(ctx, navnode)
     navmenu = render_navmenu(ctx, navnode)
     article = render_article(ctx, navnode)
 
     htmldoc = DOM.HTMLDocument(
         html[:lang=>"en"](
             head,
-            body(navmenu, article)
+            body(topbar, div[".main"](div[".container"](navmenu, article)))
```

Comment:
I would prefer avoiding unnecessary `div`s. They're not used in the style, so as far as I can tell they could be dropped. A `body(topbar, navmenu, article)` ought to do.